bin/get_nam_data.pl

```diff
@@ -30,6 +30,7 @@
 use Date::Calc;
 use File::Basename;
 use File::Copy 'move';
+use File::Copy 'copy';
```

Comment:
```suggestion
use File::Copy qw/move copy/;
```

Please accept this particular suggestion. This is how we combine more than 1 method we wish to export from the `use`'d module.